models/User.js

```diff
@@ -0,0 +1,34 @@
+/** @module models/User
+* The User Model
+* Schema:
+* username        String      username Required
+* password        String      password Required
+* email						String		  email Required
+* freelancer 			ObjectID 		freelancer profile linked to this user
+*
+* _id (ObjectID) will be added automatically by mongoose if not specified
+*/
+
+'use strict';
+const mongoose = require('mongoose');
+const ObjectID = mongoose.Schema.Types.ObjectId;
+require ('./Freelance');
+
+const User = exports.User = new mongoose.Schema({
+    username    : { type: String, required: true },
+    password    : { type: String, required: true },
+    email				: { type: String, required: true },
+    freelancer	: { type: ObjectID, ref: 'Freelance'},
```

Comment:
Even though we idealized that way, I don't think it will ever implemented for a User to have multiple freelancer profiles. So I am not too incline to change it since i sense more complications coming up for the tests that way.